ui-frontend/packages/catalog-ui-search/src/main/webapp/js/ResultUtils.js

```diff
@@ -16,9 +16,9 @@ const _ = require('underscore')
 const metacardDefinitions = require('../component/singletons/metacard-definitions.js')
 
 module.exports = {
-  refreshResult(result) {
+  refreshResult(result, metacardProperties) {
     const id = result.get('metacard').id
```

Comment:
I know this is outside your changes, but why do we do this id declaration?  🤔

Reply:
was wondering this as well :shrug: